go/staking/api/slashing.go

```diff
@@ -11,18 +11,33 @@ import (
 type SlashReason uint8
 
 const (
-	// SlashConsensusEquivocation is slashing due to equivocation.
+	// SlashConsensusEquivocation is slashing due to double signing.
```

Comment:
Hm?

Reply:
reverted this change (snuck in here after extracting these changes into the separate general changes PR)